src/implementations/vanilla/components/form/_form.scss

```diff
@@ -3,19 +3,10 @@
  */
 
 @use 'sass:map';
-@use '@ecl/theme-dev/theme';
-
-$_color-help: null !default;
-$_color-invalid: null !default;
-$_color-label: null !default;
-$_color-optional: null !default;
-$_color-required: null !default;
-$_first-element-margin: null !default;
-$_font-help: null !default;
-$_font-invalid: null !default;
-$_font-optional: null !default;
-$_form-element-margin: null !default;
-$_margin-invalid: null !default;
+
+// Exposed variables
+$theme: null !default;
+$form-group: null !default;
 
 .ecl-form-group {
```

Comment:
this css is critical, we've been adding code here to handle all the exceptions, but here we use lots of those selectors that we actually shouldn't use to decouple a bit the html structure from the styles.
Not sure we can get rid of this approach, but to a certain extent it feels this could be simplified, when really needed exceptions could be added at the component level, instead of here in the form group.  possibly style its elements regardles